src/System.Private.ServiceModel/tools/setupfiles/BuildWCFSelfHostedService.cmd

```diff
@@ -0,0 +1,49 @@
+@echo off
```

Comment:
This script should live in tools/scripts folder as test asset structure we discussed.